pkgs/by-name/ul/ultraleap-hand-tracking-control-panel/package.nix

```
@@ -0,0 +1,41 @@
+{ lib
+, stdenvNoCC
+, fetchurl
+, dpkg
+, makeWrapper
+, breakpointHook
+, autoPatchelfHook
+, libgcc
+, zlib
+}:
+stdenvNoCC.mkDerivation {
+  pname = "ultraleap-hand-tracking-control-panel";
+  version = "3.4.1";
+
+  src = fetchurl {
+    url = "https://repo.ultraleap.com/apt/pool/main/u/ultraleap-hand-tracking-control-panel/ultraleap-hand-tracking-control-panel_1125862.deb";
+    hash = "sha256-Pli0ENEtzBQGvfo2nIf09b62Xt3gzML/QYXMwtrD+7Y=";
+  };
+
+  nativeBuildInputs = [ autoPatchelfHook dpkg breakpointHook makeWrapper ];
+  buildInputs = [ libgcc zlib ];
+
+  dontBuild = true;
+
+  installPhase = ''
+    runHook preInstall
+
+    mkdir -p $out/bin $out/share $out/opt
+    cp -Tr usr/share $out/share
+    cp -Tr opt $out/opt
+
+    makeWrapper $out/opt/ultraleap-hand-tracking-control-panel/ultraleap-hand-tracking-control-panel $out/bin/ultraleap-hand-tracking-control-panel
+
+    runHook postInstall
+  '';
+
+  meta = with lib; {
+    platforms = [ "x86_64-linux" ];
+    maintainers = with maintainers; [ Scrumplex ];
+  };
```

```suggestion
  meta = with lib; {
    description = "Ultraleap Hand Tracking Control Panel";
    license = licenses.unfree;
    platforms = [ "x86_64-linux" ];
    maintainers = with maintainers; [ Scrumplex ];
  };
```
Missing metadata (license is required).